Sources/Views/PricingV2.swift

```diff
@@ -120,118 +120,11 @@ public struct PricingV2: HTML {
     }
 
     WhatToExpect()
-    PricingFAQ()
+    FAQModule(faqs: .allFaqs)
```

Comment:
moved a bunch of stuff in here to the SharedComponents directory and generalized it a bit.